src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier supplier = new ColorSupplier();
+    private final Random randNum = new Random();
+
+    public Ball getRandomBall() {
+        Ball randBall = new Ball();
+        randBall.setColor(supplier.getRandomColor());
+        randBall.setNumber(randNum.nextInt(100));
```

Comment:
100 is a magic number.
Avoid direct use of magic numbers instead you should always make it constant with self-descriptive name.

```suggestion
        randBall.setNumber(randNum.nextInt(100));
```